nominal/nominal.py

```diff
@@ -130,7 +130,7 @@ def __upload_file(self, overwrite: bool) -> requests.Response:
             self.filename = default_filename("DATASET")
 
         csv_file_buffer = io.BytesIO()
-        self.write_csv(csv_file_buffer)
+        self.write_csv(csv_file_buffer, datetime_format=r"%Y-%m-%dT%H:%M:%S.%fZ")
```

Comment:
Ensures that timestamps will always be written with the same format. In the future this should be restricted to the specific column of interest, but this works for now.